### PR TITLE
fix/32390/disputes

### PR DESCRIPTION
**Pull request by massimez**

### CODE REVIEW

File: pages/disputes/index.vue [hunk 1]
```diff
@@ -16,50 +16,53 @@
             v-for="(item, i) in disputes"
             :key="i"
             :data-selector="`TO-DISPUTES-ROUTE-${i}`"
-            class="page__card"
-            @click="toDisputes(item.id)"
+            class="wrapperCard"
```
Comment (matykoks) [inline pages/disputes/index.vue:19]: make component of CardDispute and add skeletons

File: pages/disputes/index.vue [hunk 2]
```diff
@@ -240,61 +119,21 @@ export default {
...
-    margin: 20px 0;
-    border-left: 1px solid $black0;
-    height: auto;
-  }
+
   .pager {
     margin-top: 20px;
   }
 }
+.wrapperCard{
```
Comment (matykoks) [inline pages/disputes/index.vue:134]: cleare this trash and check classes under this row

File: components/app/pages/disputes/CardDispute.vue [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+          class="cardDispute__text"
+        >
+          {{ $t('disputes.decision') }}
+          <div class="cardDispute__decision">
+            {{ dispute.decisionDescription ? dispute.decisionDescription : '-' }}
+          </div>
+        </div>
+      </div>
+    </div>
+    <div
```
Comment (matykoks) [inline components/app/pages/disputes/CardDispute.vue:51]: del this wrapper

File: pages/disputes/index.vue (hunk 2 above)
Comment (massimez) [reply pages/disputes/index.vue:134]: It's used to separate the rating stars from the card

Comment (matykoks) [reply pages/disputes/index.vue:134]: you didnt use it in layout, check it

File: components/app/pages/disputes/CardDispute.vue (hunk 3 above)
Comment (matykoks) [reply components/app/pages/disputes/CardDispute.vue]: i understand, that here you need it for container in template

File: pages/disputes/index.vue (hunk 2 above)
Comment (matykoks) [reply pages/disputes/index.vue:134]: if you can not understand about what i am texting) ask me

**Comment by matykoks** [comment]
waiting clear from trash `pages/disputes/index.vue`  (rows 159-164)
and moving `components/app/pages/disputes/CardDispute.vue` in `components/app/CardDispute.vue`

**Pull request merged**